src/tmxrasterizer/tmxrasterizer.cpp

```diff
@@ -173,3 +191,71 @@ int TmxRasterizer::render(const QString &mapFileName,
 
     return 0;
 }
+
+int TmxRasterizer::renderWorld(const QString &worldFileName,
+                               const QString &imageFileName)
+{
+    WorldManager &worldManager = WorldManager::instance();
+    QString errorString;
+    const World *world = nullptr;
+    if (worldManager.loadWorld(worldFileName, &errorString)){
+        auto const &worlds = worldManager.worlds();
+        auto WorldIt = worlds.find(worldFileName);
+        if (WorldIt != worlds.end())
+            world = WorldIt.value();
+    }
+    if (world == nullptr) {
+        qWarning("Error loading the world file \"%s\":\n%s",
+                 qUtf8Printable(worldFileName),
+                 qUtf8Printable(errorString));
+        return 1;
+    }
+        
+    auto const maps = world->allMaps();
+    QRect worldBoundingRect;
+    bool boundsAreInit = false;
+    for (const World::MapEntry &mapEntry : maps) {
+        if (!boundsAreInit){
+            boundsAreInit = true;
+            worldBoundingRect = mapEntry.rect;
+            continue;
+        }
+        if (worldBoundingRect.left() > mapEntry.rect.left())
+            worldBoundingRect.setLeft(mapEntry.rect.left());
+        if (worldBoundingRect.top() > mapEntry.rect.top())
+            worldBoundingRect.setTop(mapEntry.rect.top());
+        if (worldBoundingRect.right() < mapEntry.rect.right())
+            worldBoundingRect.setRight(mapEntry.rect.right());
+        if (worldBoundingRect.bottom() < mapEntry.rect.bottom())
+            worldBoundingRect.setBottom(mapEntry.rect.bottom());
+    }   
+        
+    QSize mapSize = worldBoundingRect.size();
+    QPoint mapOffset = worldBoundingRect.topLeft();
+    
+    QImage image(mapSize * mScale, QImage::Format_ARGB32);
```

Comment:
I understand `mTileSize` will be hard to support, since we can't even rely on each map using the same tile size, but supporting `mSize` would be useful (it could even enable world thumbnails).